src/tv2-common/helpers/config.ts

```diff
@@ -17,11 +17,12 @@ export function findGfxSetup<ShowStyleConfig extends TV2ShowstyleBlueprintConfig
 	config: ShowStyleConfig,
 	fallbackGfxSetup: ShowStyleConfig['GfxSetups'][0]
 ): ShowStyleConfig['GfxSetups'][0] {
+	const defaultSetupName = config.GfxDefaults[0].DefaultSetupName
 	const foundTableConfigGfxSetup: TableConfigGfxSetup | undefined = config.GfxSetups.find(
-		(tableConfigGfxSetup) => tableConfigGfxSetup.Name === config.GfxDefaults[0].GfxSetup
+		(tableConfigGfxSetup) => tableConfigGfxSetup.Name === defaultSetupName?.label
```

Comment:
I realized `_id` and `value`  should be used here, as the label may get out of sync

Reply:
Is it possible to do so when `tableConfigGfxSetup` only exposes the config values?
But yeah, as mentioned in https://github.com/tv2/tv-automation-server-core/pull/232, there is for sure the issue of desync. Fortunately Jacob is aware of this at least.